src/Components/DnDLayout/GridTile.tsx

```diff
@@ -204,7 +204,7 @@ const GridTile = ({ widgetType, isDragging, setIsDragging, setWidgetAttribute, w
                   <Button
                     className="pf-v5-u-font-weight-bold pf-v5-u-font-size-xs pf-v5-u-p-0"
                     variant="link"
-                    onClick={() => window.open(headerLink.href, '_blank')}
+                    onClick={() => window.open(headerLink.href, '_self')}
```

Comment:
Instead of reloading the window, can we use the router link? We can check for the href origin, and if it does not exist or if it matches the current origin, we can safely use the router link instead of using the window methods.